src/show/pages/public/CreateProject/index.js

```diff
@@ -0,0 +1,81 @@
+import axios from 'axios';
+import { useState } from 'react';
+import StepWizard from 'react-step-wizard';
+import { useNavigate } from 'react-router-dom';
+import { PROJECT_PATH } from '../../../../process/routes/paths';
```

Comment:
This should be grouped with the one bellow since it's not imported from a library.

Reply:
missclick

Reply:
done